src/exml_query.erl

```diff
@@ -21,16 +21,24 @@
 -export([cdata/1]).
 
 -type element_with_ns() :: {element_with_ns, binary()}.
--type element_with_name_and_ns () :: {element_with_ns, binary(), binary()}.
--type element_with_attr_of_value () :: {element_with_attr, binary(), binary()}.
+-type element_with_name_and_ns() :: {element_with_ns, binary(), binary()}.
+-type element_with_attr_of_value() :: {element_with_attr, binary(), binary()}.
+-type path() :: [cdata |
+                 {element, binary()} |
+                 {attr, binary()} |
+                 element_with_ns() |
+                 element_with_name_and_ns() |
+                 element_with_attr_of_value()].
 
--type path() :: [cdata | %% selects cdata from the element
-                 {element, binary()} | % selects subelement with given name
-                 {attr, binary()} | % selects attr of given name
-                 element_with_ns() | % selects subelement with given namespace
-                 element_with_name_and_ns() | % selects subelement with given name and namespace
-                 element_with_attr_of_value() % selects subelement with given attribute and value
-                ].
+                                                %% selects cdata from the element
+                                                % selects subelement with given name
+
+                                    % selects attr of given name
+ % selects subelement with given namespace
+
+                                              % selects subelement with given name and namespace
+
+ % selects subelement with given attribute and value
```

Comment:
Looks like the formatting put all the comments after the type specification. I wonder if anything can be done about it.

Reply:
Check my other comment. It's a known issue, sadly. One that we tried to solve but couldn't.
If you feel like taking a deep dive into rebar3_format and submitting a PR for this, we would gladly accept it :)